examples/schelling.jl

```diff
@@ -327,3 +327,86 @@ adf
 
 # We nicely see that the larger `:min_to_be_happy` is, the slower the convergence to
 # "total happiness".
+
+# ## Saving the model state

```

Comment:
Can move this above distributed computing and ensembles, it's a simpler thing to learn and will probably be used by more people.